docs/guide/custom-ui.md

```diff
@@ -0,0 +1,62 @@
+# Customizing the Apostrophe user interface
+
+This guide focuses on how to customize Apostrophe's administrative user interface, or "admin UI." The built-in functionality covers most situations, but sometimes you'll want to add or change functionality.
+
+## Registering custom field types
+
+Apostrophe's [schema field types](content-schema.md) cover many situations, but you might wish to add a new one.
+
+## Creating custom Vue components
```

Comment:
I need to move this after the "by name" material because it is the exception to that rule in this particular dev story.